includes/class-dispatcher.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * ActivityPub Dispatcher Class.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub;
+
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Followers;
+use Activitypub\Transformer\Factory;
+
+/**
+ * ActivityPub Dispatcher Class.
+ *
+ * @author Matthias Pfefferle
+ *
+ * @see https://www.w3.org/TR/activitypub/
+ */
+class Dispatcher {
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action( 'activitypub_process_outbox', array( self::class, 'process_outbox' ), 10, 1 );
```

Comment:
```suggestion
		\add_action( 'activitypub_process_outbox', array( self::class, 'process_outbox' ) );
```
Priority 10 and 1 accepted arg are [defaults](https://developer.wordpress.org/reference/functions/add_action/).